TDOAuth.m

```diff
@@ -371,7 +371,7 @@ + (NSURLRequest *)URLRequestForPath:(NSString *)unencodedPathWithoutQuery
             if (error || !postbody) {
                 NSLog(@"Got an error encoding JSON: %@", error);
             } else {
-                [oauth setParameters:@{}]; // empty dictionary populates variables without putting data into the signature_base
+                [oauth setParameters:@[]]; // empty dictionary populates variables without putting data into the signature_base
```

Comment:
I see that the old code was wrong, but the code no longer matches the comment. Can you make those agree?